Gems/AtomTressFX/Code/Rendering/SharedBufferInterface.h

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/smart_ptr/intrusive_ptr.h>
+#include <AzCore/std/smart_ptr/intrusive_base.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/EBus/Event.h>
+
+#include <AtomCore/Instance/Instance.h>
+#include <Atom/RHI/Allocator.h>
+
+namespace AZ
+{
+    namespace RHI
+    {
+        class Buffer;
+        class BufferView;
+    }
+
+    namespace RPI
+    {
+        class BufferAsset;
+        class Buffer;
+    }
+
+    namespace Render
+    {
+        class SharedBufferAllocation;
+
+        //! A class for allocating memory for skinning buffers
+        class SharedBufferInterface
+        {
+        public:
+            AZ_RTTI(AZ::Render::SharedBufferInterface, "{3CCB13CB-16FF-43F5-98DC-F36B2A9F8E5E}");
+
+            SharedBufferInterface()
+            {
+                Interface<SharedBufferInterface>::Register(this);
+            }
+
+            virtual ~SharedBufferInterface()
+            {
+                Interface<SharedBufferInterface>::Unregister(this);
+            }
+
+            static SharedBufferInterface* Get()
+            {
+                return Interface<SharedBufferInterface>::Get();
+            }
+
+            //! Returns the buffer asset that is used for all skinned mesh outputs
```

Comment:
Anton s right, but since I checked and this is not used I decided to remove it for now and re-implement if ever needed.
I also used this opportunity to remove the two remarked lines below.
tnx.